examples/todo-list/src/models/author.model.ts

```diff
@@ -0,0 +1,18 @@
+import {Entity, model, property, belongsToUniquely} from '@loopback/repository';
+import {TodoList} from './todo-list.model';
+
+@model()
+export class Author extends Entity {
+  @belongsToUniquely(() => TodoList)
```

Comment:
This scenario does not make sense to me. In a typical application, a single author has many todos.

What are the arguments for using Author instead of other alternatives you have considered?

Reply:
Well I was thinking of "a todo-list can only have one owner/author" and thus this model. But yea, as discussed, I'm going to revert to `todoListImage` model and describe the limitation for the binary data property in a follow-up issue and leave it as string for now.